NASA-PDS/validate
# issue 47: Add flag to temporarily disable context validation

Added flag "--no-context-valid" to temporarily disable context validation

To test:
**bin/validate  src/test/resources/github47/test_context_products.xml --no-context-valid**
_WARNING  [error.label.context_ref_not_found]   line 29: 'urn:nasa:pds:context:investigation:mission.mars_exploration_rover::null::null' was found  by LidVid 'urn:nasa:pds:context:investigation:mission.mars_exploration_rover' in the supplied list of registered context products. But there are no name and/or type.
      WARNING  [error.label.context_ref_not_found]   line 40: 'urn:nasa:pds:context:instrument_host:spacecraft.FOO' could not be found in the supplied list of registered context products.
      WARNING  [error.label.context_ref_not_found]   line 49: 'urn:nasa:pds:context:instrument_host:spacecraft.mer2' was found in the supplied list of registered context products.but name 'MER2' and 'Spacecraft' are different with that in the supplied list of registered context products.
      WARNING  [error.label.context_ref_not_found]   line 58: 'urn:nasa:pds:context:instrument:pancam.mer2' was found in the supplied list of registered context products.but name 'Pancam' and 'Instrument' are different with that in the supplied list of registered context products._
**bin/validate  src/test/resources/github47/test_context_products.xml**
Will see those in the output(report):
_ ERROR  [error.label.context_ref_not_found]   line 29: 'urn:nasa:pds:context:investigation:mission.mars_exploration_rover::null::null' was found  by LidVid 'urn:nasa:pds:context:investigation:mission.mars_exploration_rover' in the supplied list of registered context products. But there are no name and/or type.
      ERROR  [error.label.context_ref_not_found]   line 40: 'urn:nasa:pds:context:instrument_host:spacecraft.FOO' could not be found in the supplied list of registered context products.
      ERROR  [error.label.context_ref_not_found]   line 49: 'urn:nasa:pds:context:instrument_host:spacecraft.mer2' was found in the supplied list of registered context products.but name 'MER2' and 'Spacecraft' are different with that in the supplied list of registered context products.
      ERROR  [error.label.context_ref_not_found]   line 58: 'urn:nasa:pds:context:instrument:pancam.mer2' was found in the supplied list of registered context products.but name 'Pancam' and 'Instrument' are different with that in the supplied list of registered context products._
